python/dolfinx/wrappers/la.cpp

```diff
@@ -96,8 +96,11 @@ void la(py::module& m)
   // dolfinx::la::Vector
   py::class_<dolfinx::la::Vector<PetscScalar>,
              std::shared_ptr<dolfinx::la::Vector<PetscScalar>>>(m, "Vector")
-      .def_property_readonly("array",
-                             &dolfinx::la::Vector<PetscScalar>::mutable_array);
+      .def_property_readonly(
```

Comment:
Keep the reference version - it's simpler and consistent with what we do elsewhere.